rules/macos/defense_evasion_attempt_del_quarantine_attrib.toml

```diff
@@ -4,23 +4,23 @@ integration = ["endpoint"]
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
-updated_date = "2023/11/15"
+updated_date = "2024/02/06"
 
 [rule]
 author = ["Elastic"]
 description = """
-Identifies a potential Gatekeeper bypass. In macOS, when applications or programs are downloaded from the internet,
-there is a quarantine flag set on the file. This attribute is read by Apple's Gatekeeper defense program at execution
-time. An adversary may disable this attribute to evade defenses.
+Detects deletion of the quarantine attribute by an unusual process (xattr). In macOS, when applications or programs are downloaded from 
+the internet, there is a quarantine flag set on the file. This attribute is read by Apple's Gatekeeper defense program at execution time. 
+An adversary may disable this attribute to evade defenses.
 """
 from = "now-9m"
 index = ["logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
-name = "Attempt to Remove File Quarantine Attribute"
+name = "Quarantine Attrib Removed by Unsigned or Unstrusted Process"
```

Comment:
Not reviewing this PR, just here to get some examples of rule name updates 😆 

But there's a typo in `Unstrusted` that might be hard to catch